surveys/2024-04/questions.md

```diff
@@ -0,0 +1,160 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+  - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+  - big tech
+  - company
+  - startup
+  - university
+  - individual
+  - other
+
+## Node.js Usage
+
+- Which groups do you identify with? (multiple choice)
+  - Direct end users: Users who run tools themselves
+  - Application operators: Service and infrastructure providers
+  - Application developers: Frontend tools consumers, backend server authors, tools authors
+  - Library & package authors: Users who write libraries and packages to be included on other applications
+  - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+  - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+
+- Are you part of a group not covered ? If yes which one ? (open question)
+
+- What is your use cases of Node.js ? (if multiple use case, separate with a comma) (open question)
```

Comment:
Change to bullet point considering the answer from last year: https://github.com/nodejs/next-10/blob/main/surveys/2023-04/Node.js%20Survey%20open%20ended%20answers.pdf

- cli tools
- testing
- etc...
TODO